apps/settings/elements/hotspot.html

```diff
@@ -13,25 +13,13 @@ <h2 data-l10n-id="internetSharing-wifi"></h2>
         <li>
           <gaia-switch id="tethering-wifi-enabled">
             <label data-l10n-id="wifi-hotspot"></label>
+            <details hidden id="wifi-hotspot-enabled-msg"></details>
+            <details hidden id="wifi-hotspot-disabled-msg" data-l10n-id="wifi-hotspot-disabled-msg"></details>
```

Comment:
please use single `wifi-hotspot-msg` and use `setAttribute('data-l10n-id', string)` for enabled / `removeAttribute('data-l10n-id')` for disabled cases